components/match2/wikis/clashroyale/match_group_display_bracket_custom.lua

```diff
@@ -0,0 +1,43 @@
+---
+-- @Liquipedia
+-- wiki=clashroyale
+-- page=Module:MatchGroup/Display/Bracket/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+
+local BracketDisplay = Lua.import('Module:MatchGroup/Display/Bracket', {requireDevIfEnabled = true})
+local MatchGroupUtil = Lua.import('Module:MatchGroup/Util', {requireDevIfEnabled = true})
+
+local CustomBracketDisplay = Table.copy(BracketDisplay)
+
+function CustomBracketDisplay.BracketContainer(props)
+	local bracket = MatchGroupUtil.fetchMatchGroup(props.bracketId)
+	return BracketDisplay.Bracket({
+		bracket = bracket,
+		config = Table.merge(props.config, {
+			opponentHeight = CustomBracketDisplay.computeBracketOpponentHeight(bracket.matchesById),
+		})
```

Comment:
```suggestion
		config = Table.merge(props.config, {
			opponentHeight = math.max(CustomBracketDisplay.computeBracketOpponentHeight(bracket.matchesById), props.config.opponentHeight),
		})
```
Allows for manually setting larger than minimum sizes still

Reply:
would need some default value for empty `props.config.opponentHeight` else it errors